docs/index.md

```diff
@@ -24,3 +22,8 @@ See the introductory blog story and three status updates for a longer introducti
   {% endif %}
 {% endfor %}
 
+## See also
+
+* [Introductory blog story](https://0pointer.de/blog/projects/systemd.html)
```

Comment:
The `https` version of this link does not work, at least from here. The `http` version does.